shopelectro/selenium/pages/category.py

```diff
@@ -0,0 +1,27 @@
+import typing
+
+from shopelectro.selenium.elements import ProductCard
+from shopelectro.selenium.pages import Page
+
+from django.urls import reverse
+
+# @todo #682:60m Implement and reuse shopelectro.selenium.CategoryPage for selenium tests.
+
+
+class CategoryPage(Page):
+
+    def __init__(self, driver, slug):
+        super().__init__(driver)
+        self.slug = slug
+
+    @property
+    def address(self):
```

Comment:
before this moment we used `path` for reversed part of url without domain. Why `address` now?
Maybe we'll leave `path`?

OrderPage too